src/cli/commands/check.js

```diff
@@ -119,7 +119,7 @@ export async function verifyTreeCheck(
         const relative = path.relative(cwd, subDepPath);
         const locations = path.normalize(relative).split(registry.folder + path.sep).filter(dir => !!dir);
         locations.pop();
-        while (locations.length >= 0) {
+        while (locations.length) {
```

Comment:
After giving it a second look this is incorrect - `>=` returns true on an empty array, but your change returns false. I think that's why the tests are failing 🙂

Reply:
Ok, I'll revert this one.